libkbfs/journal_md_ops.go

```diff
@@ -207,10 +207,10 @@ func (j journalMDOps) GetForHandle(
 	// If the journal has a head, use that.
 	irmd, err := j.getHeadFromJournal(
 		ctx, tlfID, NullBranchID, mStatus, handle)
-	if err != nil {
+	if err != nil && err != errTLFJournalDisabled {
```

Comment:
Might be cleaner to put this check in `getHeadFromJournal`, since that is supposed to encapsulate the journal stuff